statesman-server/src/main/java/io/appform/statesman/server/dao/message/MessageConstructor.java

```diff
@@ -0,0 +1,60 @@
+package io.appform.statesman.server.dao.message;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.appform.statesman.model.MessageConfig;
+import io.appform.statesman.model.exception.ResponseCode;
+import io.appform.statesman.model.exception.StatesmanError;
+import io.appform.statesman.server.utils.MapperUtils;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import java.util.Optional;
+
+
+@Slf4j
+public class MessageConstructor implements IMessageConstructor {
+
+    private final MessageConfigStoreCommand messageConfigStoreCommand;
+    private final String DEFAULT_FILED_NAME = "default";
+
+    @Inject
+    public MessageConstructor(MessageConfigStoreCommand messageConfigStoreCommand) {
+        this.messageConfigStoreCommand = messageConfigStoreCommand;
+    }
+
+
+    @Override
+    public String constructMessage(String messageId, String language, String state) {
+
+        Optional<MessageConfig> optionalMessageConfig = messageConfigStoreCommand.get(messageId);
+        if(!optionalMessageConfig.isPresent()){
+            throw new StatesmanError("No message config found", ResponseCode.NO_PROVIDER_FOUND);
+        }
+
+        MessageConfig storeOutput = optionalMessageConfig.get();
```

Comment:
can we add a comment of the json structure ? will help whoever will read this code in future

Reply:
done